backend/js/src/main/scala/aqua/backend/js/JavaScriptService.scala

```diff
@@ -64,16 +66,21 @@ case class JavaScriptService(srv: ServiceRes) {
       |        service = args[2];
       |    }
       |
-      |      peer.internals.callServiceHandler.use((req, resp, next) => {
-      |          if (req.serviceId !== serviceId) {
-      |              next();
-      |              return;
-      |          }
-      |  
-      |          ${fnHandlers}
-      |  
-      |          next();
-      |      });
+      |    let incorrectServiceDefinitions = missingFields(service, [${membersNames.map { n => s"'$n'"}.mkString(", ")}]);
```

Comment:
It's better to use `const` for every declaration which is not going to be reassigned